tests/api/libraries/test_libraries_rest.py

```diff
@@ -173,12 +173,14 @@ def test_library_never_open(lib_sion):
 
 
 def test_library_can_delete(lib_martigny, librarian_martigny,
-                            loc_public_martigny):
+                            loc_public_martigny, acq_receipt_fiction_martigny):
     """Test can delete a library."""
     can, reasons = lib_martigny.can_delete
     assert not can
+    assert 'links' in reasons
```

Comment:
Maybe you can delete this, because `assert reasons['links']['locations']` will fail if `links` is missing.